### PR TITLE
vault target

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/11723

We're only scrapping the vault node here, so node-exporter and cert-exporter which sits on the vault node, not vault itself.

image

### CODE REVIEW

**Review by mig4 — APPROVED** [review]

**Comment by JosephSalisbury** [comment]
is the target on 10300 a node-exporter?

File: files/templates/additional-scrape-configs.template.yaml [hunk 1]
```diff
@@ -1875,4 +1875,23 @@
...
+- job_name: {{ .ClusterID }}-prometheus/vault-{{ .ClusterID }}/0
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+  - targets:
+    - {{ .Vault }}:10300
+    labels:
+      app: vault
+  - targets:
+    - {{ .Vault }}:9005
```
Comment (omegas27) [inline files/templates/additional-scrape-configs.template.yaml:1887]: What is this cert exporter about? Do we not have a cert exporter app?

Comment (TheoBrigitte) [reply files/templates/additional-scrape-configs.template.yaml:1887]: There's a cert-exporter app running on the cluster, but there's an additional cert-exporter on the vault node.

**Comment by TheoBrigitte** [comment]
I took the config from g8s-prometheus: https://github.com/giantswarm/g8s-prometheus/blob/e927fe8e3f17354b6ee884badd39f15cf0d9840e/helm/g8s-prometheus/templates/prometheus/prometheus-configmap.yaml#L433-L445

> is the target on 10300 a node-exporter?

yes, 10300 is node-exporter

**Review by omegas27 — APPROVED** [review]

**Comment by JosephSalisbury** [comment]
my thinking was if it's a node-exporter, `app='node-exporter'` would make more sense, no?

**Pull request merged**